cro-clib/main.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
```

Comment:
probably move this to `example` directory inside this crate

Reply:
yes